server/src/main/java/org/elasticsearch/search/aggregations/pipeline/InternalPercentilesBucket.java

```diff
@@ -77,6 +80,7 @@ protected void doWriteTo(StreamOutput out) throws IOException {
         out.writeNamedWriteable(format);
         out.writeDoubleArray(percentiles);
         out.writeDoubleArray(percents);
+        out.writeBoolean(keyed);
```

Comment:
Similar for writing values, need to make sure we only send `keyed` if the node we're talking to supports it:

```
if (out.getVersion().onOrAfter(Version.V_7_0_0)) {
  out.writeBoolean(keyed);
}
```